Emby.Dlna/Didl/DidlBuilder.cs

```diff
@@ -763,37 +782,37 @@ private void WriteObjectClass(XmlWriter writer, BaseItem item, StubType? stubTyp
 
         private void AddPeople(BaseItem item, XmlWriter writer)
```

Comment:
How do you know this method is safe to uncomment?

Reply:
We have nightlies for a reason ;)
And the person that did the .net core port removed random pieces of code